modular_zubbers/code/datums/quirks/positive_quirks/defensive.dm

```diff
@@ -0,0 +1,37 @@
+/datum/quirk/defensive
+	name = "Jerk"
+	desc = "You are rude and impatient, and want people to get out of your way. "
+	icon = FA_ICON_FROWN_OPEN
+	medical_record_text = "Patient has anger issues."
+	value = 6
```

Comment:
kinda a huge value for something not that incredibly negative

Reply:
I was watching him code it in a stream. This is a justified value imo